internal/app.go

```diff
@@ -260,6 +263,11 @@ func (a *App) cleanEvent(event *ics.VEvent) {
 			description = location + "\n" + description
 			event.SetLocation(building)
 		}
+		if roomID := reNavigaTUM.FindString(location); roomID != "" {
+			roomID = strings.Trim(navigaTUMURL, "()")
+			description = fmt.Sprintf("https://nav.tum.de/room/%s\n%s", roomID, description)
+			description = navigaTUMURL + "\n" + description
```

Comment:
Previous suggestion from me was not fault-free

```suggestion
			roomID = strings.Trim(roomID, "()")
			description = fmt.Sprintf("https://nav.tum.de/room/%s\n%s", roomID, description)
```